src/cmd-buildextend-installer

```diff
@@ -78,6 +79,14 @@ builddir = builds.get_build_dir(args.build)
 buildmeta_path = os.path.join(builddir, 'meta.json')
 buildmeta = GenericBuildMeta(workdir=workdir)
 
+# used to lock
+build_semaphore = os.path.join(
```

Comment:
Right, this is what I meant by "lock the whole build".  But why not use the locking API you added?

Reply:
I hit a bug in the lifetime param of the `flufl.lock` that I need to dig on in the Fedora Package for `flufl.lock.` I intended on using the Lock API over this. I anticipate that once I figure out the problem with `flufl.lock` I'll PR a change.

Reply:
https://github.com/darkmuggle/coreos-assembler/commit/93f89885d15d11912d1b548fdfc538350db0a50f is the WIP.